hack/overrides/jaeger.yml

```diff
@@ -7,6 +7,8 @@
 #
 version: '3'
 
+volumes:
+  elasticsearch-data:
```

Comment:
not a big deal, more of a question: is the need of a named volume volume in order to have data being persisted across restarts of elasticsearch?

Reply:
yup, don't want those yellow indices
<img width="1109" alt="Screen Shot 2020-04-17 at 9 39 05 AM" src="https://user-images.githubusercontent.com/22056320/79575138-543a3180-808f-11ea-997a-3bd3405bd3e5.png">

Reply:
it sure is -- i guess that image doesn't have a volume by default